src/AppProviders.tsx

```diff
@@ -1,5 +1,6 @@
 import { ChakraProvider, ColorModeScript } from '@chakra-ui/react'
 import { DefiManagerProvider } from 'features/defi/contexts/DefiManagerProvider/DefiManagerProvider'
+import { get } from 'lodash'
```

Comment:
```suggestion
import get from 'lodash/get'
```

this will tree shake better